packages/server/graphql/mutations/helpers/getReflectionEntities.ts

```diff
@@ -1,26 +1,36 @@
-import getGoogleLanguageManager from '../../../getGoogleLanguageManager'
-import addLemmaToEntities from './autoGroup/addLemmaToEntities'
-import sanitizeAnalyzedEntitiesResponse from './autoGroup/sanitizeAnalyzedEntititesResponse'
-import manageGoogleNLPErrorResponse from './manageGoogleNLPErrorResponse'
+import {GoogleAnalyzedEntity} from '../../public/resolverTypes'
 
-const getReflectionEntities = async (plaintextContent: string) => {
+const getBasicLemma = (word: string): string => {
```

Comment:
-2 this is just too brittle. it'll only work on english, and even then words like "bread" and "Brent" are gonna get recognized as the same lemma. In cases like this, where there's a high likelihood of bad data, we're usually better of with just no data at all.